recipes/pysmithchart/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "pysmithchart" %}
+{% set version = "0.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pysmithchart-{{ version }}.tar.gz
+  sha256: 9b10439a9b0e9883fac29308bfb5d78e18fca978197f52d5bc19efb436ef1cc6
+
+build:
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv"
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python {{ python_min }}
+    - setuptools
+  run:
+    - matplotlib-base
+    - numpy
+    - python {{ python_min }}
```

Comment:
```suggestion
    - python >={{ python_min }}
```

Reply:
ack.  sorry, made some of your suggested changes already before I saw your comments.